packages/mdc-snackbar/mdc-snackbar.scss

```diff
@@ -69,6 +69,10 @@
     }
   }
 
+  &[aria-hidden] {
```

Comment:
Why not select on `.mdc-snackbar` and `.mdc-snackbar--active` for this?

Reply:
Done!

Reply:
Changes done as discussed!